engineering/documents/CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing Guidelines
+
+source{d} go-git project is [Apache 2.0 licensed](LICENSE) and accept
+contributions via GitHub pull requests.  This document outlines some of the
+conventions on development workflow, commit message formatting, contact points
+and other resources to make it easier to get your contribution accepted.
+
+## Certificate of Origin
+
+By contributing to this project you agree to the Developer Certificate of
+Origin (DCO). This document was created by the Linux Kernel community and is a
+simple statement that you, as a contributor, have the legal right to make the
+contribution. See the [DCO](DCO) file for details.
```

Comment:
The DCO file is to be the same and present all across our repos, right?

Reply:
yep is the same for all of them

Reply:
It would also be easier to find it after #76 ;)